staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3667,9 +3667,9 @@ type PodSpec struct {
 	// +optional
 	AutomountServiceAccountToken *bool `json:"automountServiceAccountToken,omitempty" protobuf:"varint,21,opt,name=automountServiceAccountToken"`
 
-	// NodeName is a request to schedule this pod onto a specific node. If it is non-empty,
-	// the scheduler simply schedules this pod onto that node, assuming that it fits resource
-	// requirements.
+	// NodeName indicates in which node this pod is scheduled.
+	// If empty, this pod is a candidate for scheduling by the scheduler defined in schedulerName.
+	// If it is non-empty, the kubelet for this node becomes responsible for the lifecycle of this pod.
```

Comment:
we could also mention using nodeSelector and affinity.nodeAffinity to target specific nodes while still allowing scheduling to work.

Reply:
Could I just link to the website documentation instead?

Reply:
A link out is fine, but *some* indication this is the wrong field to use if you want scheduling involved seems important

Reply:
Added